spec/statement.dd

```diff
@@ -682,6 +682,12 @@ $(CONSOLE
         }
         --------------
 
+    $(P It is important to make sure that, if $(D opApply) catches any exceptions, that those
+        exceptions did not origin at the delegate passed to $(I opApply). Catching an exception
+        that escaped from the delegate would cause a condition that the user expected would terminate
+        the loop to not do so.
```

Comment:
1. "origin at" -> originate from
2. The last sentence is cryptic. Possibly better: "The user would expect exceptions thrown from a `foreach` body to both terminate the loop, and propagate outside the `foreach` body."